ocs_ci/ocs/resources/pod.py

```diff
@@ -3079,3 +3079,35 @@ def check_ceph_cmd_execute_successfully():
 
         logger.warning(f"Failed to execute the ceph command due to the error {str(ex)}")
         return False
+
+
+def search_pattern_in_pod_logs(
```

Comment:
please move this function to some common place outside of the tests

Reply:
Can you suggest the place?

Reply:
This function not in test case file